<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	border-radius:10px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(55, 59, 90);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,5,0,7,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="TitleBar">
         <item>
          <widget class="QFrame" name="titleBarFrame">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(55, 59, 90);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="AppTitle">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Oswald Light</family>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">padding-left:5px;
color:white;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_3">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QPushButton" name="minimizeButton">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>10</y>
                 <width>17</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>17</width>
                 <height>17</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Minimize</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{
	border:none;
	border-radius:8px;
	background-color: rgb(174, 84, 252);
}
QPushButton:hover
{
	background-color: rgba(174, 84, 252,150);
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QPushButton" name="maximizeButton">
               <property name="geometry">
                <rect>
                 <x>40</x>
                 <y>10</y>
                 <width>17</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>17</width>
                 <height>17</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Maximize</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{
	border:none;
	border-radius:8px;
	background-color: rgb(174, 84, 252);
}
QPushButton:hover
{
	background-color: rgba(174, 84, 252,150);
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QPushButton" name="closeButton">
               <property name="geometry">
                <rect>
                 <x>70</x>
                 <y>10</y>
                 <width>17</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>17</width>
                 <height>17</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Close</string>
               </property>
               <property name="whatsThis">
                <string>Close</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{
	border:none;
	border-radius:8px;
	background-color: rgb(174, 84, 252);
}
QPushButton:hover
{
	background-color: rgba(174, 84, 252,150);
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="HeadAppNameContainer">
         <item>
          <widget class="QLabel" name="HeadAppName">
           <property name="font">
            <font>
             <family>Oswald</family>
             <pointsize>28</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(55, 59, 90);
color:white;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;AVES&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-weight:200;&quot;&gt;Know the birds&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="verticalFrame_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(55, 59, 90);
color:white;</string>
         </property>
         <layout class="QVBoxLayout" name="OperationSection">
          <property name="leftMargin">
           <number>100</number>
          </property>
          <property name="rightMargin">
           <number>100</number>
          </property>
          <property name="bottomMargin">
           <number>25</number>
          </property>
          <item>
           <widget class="QPushButton" name="SelectImageButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>99</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>99</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
color:white;
padding:25px;

border-radius:30;
background-color:qlineargradient(spread:repeat, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(248, 72, 193, 255), stop:1 rgba(170, 85, 255, 255))
}
QPushButton:pressed {
    background-color: qlineargradient(spread:repeat, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(194, 56, 153, 255), stop:1 rgba(133, 66, 200, 255));
}
QPushButton:disabled{
background-color:grey;
}</string>
            </property>
            <property name="text">
             <string>Select Image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="explore_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>99</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
color:white;
padding:25px;

border-radius:30;
background-color:qlineargradient(spread:repeat, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(248, 72, 193, 255), stop:1 rgba(170, 85, 255, 255))
}
QPushButton:pressed {
    background-color: qlineargradient(spread:repeat, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(194, 56, 153, 255), stop:1 rgba(133, 66, 200, 255));
}
QPushButton:disabled{
background-color:grey;
}</string>
            </property>
            <property name="text">
             <string>Explore</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="ResultSection">
         <item>
          <widget class="QFrame" name="ResultFrame">
           <layout class="QHBoxLayout" name="ResultMainBox">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>1</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="BirdImageContainer">
              <item>
               <widget class="QLabel" name="BirdImage">
                <property name="minimumSize">
                 <size>
                  <width>405</width>
                  <height>254</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>405</width>
                  <height>254</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(55, 59, 90);
color:white;
</string>
                </property>
                <property name="text">
                 <string>BirdImage</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="ResultDetails">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="ResultBirdNameContainer">
                <item>
                 <layout class="QHBoxLayout" name="ResultBirdNameHC">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <layout class="QVBoxLayout" name="BirdNameLabelContainer">
                    <item>
                     <widget class="QLabel" name="BirdNameLabel">
                      <property name="font">
                       <font>
                        <pointsize>10</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">background-color: rgb(55, 59, 90);
color:white;</string>
                      </property>
                      <property name="text">
                       <string>Bird Name:</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="BirdNameValueContainer">
                    <item>
                     <widget class="QLabel" name="BirdNameValue">
                      <property name="font">
                       <font>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">background-color: rgb(55, 59, 90);
color:white;</string>
                      </property>
                      <property name="text">
                       <string>Bird Name Here</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                      <property name="wordWrap">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QFrame" name="verticalFrame_8">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(55, 59, 90);
color:white;</string>
                </property>
                <layout class="QVBoxLayout" name="MoreInfoContainer">
                 <property name="leftMargin">
                  <number>50</number>
                 </property>
                 <property name="rightMargin">
                  <number>50</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="MoreInfoButton">
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">
QPushButton {
color:white;
padding:20px;
border:7x solid rgb(0, 183, 74);
border-radius:20;
background-color:qlineargradient(spread:repeat, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(248, 72, 193, 255), stop:1 rgba(170, 85, 255, 255));
}
QPushButton:pressed {
    background-color: qlineargradient(spread:repeat, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(194, 56, 153, 255), stop:1 rgba(133, 66, 200, 255));
}

QPushButton:disabled{
background-color:grey;
}</string>
                   </property>
                   <property name="text">
                    <string>Show Details</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="FooterSection">
         <item>
          <widget class="QLabel" name="CreditsText">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(42, 46, 70);
color:rgb(97, 97, 97);</string>
           </property>
           <property name="text">
            <string>Created by Aayush and Abhishek</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
